extension/lib/webcompat.js

```diff
@@ -67,6 +67,11 @@ class WebCompat {
   getCSSDeclarationBlockIssues(declarations, browsers) {
     let summaries = [];
     for (const { name: property, value } of declarations) {
+      // Ignore CSS variable definition from the targets we will check.
```

Comment:
I think this comment would be a little more correct and specific: 
"Ignore CSS custom properties as any name is valid"

Reply:
Okay, thanks!